content/blog/2016/branding-type.md

```diff
@@ -170,15 +176,27 @@ method of mixing and matching, but neither did we feel that any one
 element collage encapsulated our brand. We tried style tiles, but those
 felt both too specific and not specific enough.
 
-![image][1]
+{% call embed.img(
```

Comment:
Also: no need to use `call` when we're not passing anything into the block. The `embed.img` macro doesn't accept a `caller()` anyway.